device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/device/TemperatureController.java

```diff
@@ -187,14 +314,15 @@ public DeviceMethodData call(String methodName, Object methodData, Object contex
                                 .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
 
                         if (!filteredReadings.isEmpty()) {
+                            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```

Comment:
The other formatting we had earlier does not work with validation so changing to same format as C# samples

Reply:
Oh man, what a headache!